### PR TITLE
Feature/ Create input component

### PR DESCRIPTION
**Pull request by JennyDoJD**
- Design
image

- Result
image

- Ticket: [Create components](https://trello.com/c/bvrOqz8S/42-create-components)

### CODE REVIEW

File: practice/src/css/components/input.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  border: 1px solid var(--tertiary-border-color);
+  border-radius: 8px;
+  padding: 14px;
+  background-color: var(--tertiary-background-color);
+  font-size: var(--font-size-xs);
+  font-weight: var(--font-weight-regular);
+  line-height: var(--line-height-lg);
+  color: var(--primary-text-colors);
+
+  &:nth-last-child(1) {
```
Comment (huyencao-agilityio) [inline practice/src/css/components/input.css]: you would like to style for the Quantity input, right?

Comment (JennyDoJD) [reply practice/src/css/components/input.css]: I had confusion here so I fixed it

File: practice/src/css/components/input.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  border-radius: 8px;
+  padding: 14px;
+  background-color: var(--tertiary-background-color);
+  font-size: var(--font-size-xs);
+  font-weight: var(--font-weight-regular);
+  line-height: var(--line-height-lg);
+  color: var(--primary-text-color);
+}
+
+.input-quantity {
```
Comment (huyencao-agilityio) [inline practice/src/css/components/input.css]: Should create a common class here. Such as .input-form-lg, .input-form-sm. You can refer https://getbootstrap.com/docs/3.4/css/#forms-control-sizes

File: practice/src/css/components/input.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  line-height: var(--line-height-lg);
+  color: var(--primary-text-color);
+}
+
+.input-form-lg {
+  margin-bottom: 16px;
+}
+
+.input-form-sm {
+  margin-right: 127px;
```
Comment (huyencao-agilityio) [inline practice/src/css/components/input.css]: Should set margin for `form-group` instead of input

**Pull request merged**